workflow/context.go

```diff
@@ -64,7 +65,8 @@ func (wfc *WorkflowContext) CallActivity(activity interface{}, opts ...callActiv
 }
 
 // CallChildWorkflow returns a completable task for a given workflow.
-// NOTE: The task must be invoked with the .Await(output interface{}) method.
+// NOTE: The task must be invoked with the .Await(output interface{}) method, this does not need to
+// be done at call time for example when using the fan-out/fan-in pattern.
```

Comment:
I still don't feel great about this explanation. Here's my suggestion. I apologize that it's a bit verbose.
```go
// You must call Await(output any) on the returned Task to block the workflow and wait for the task to complete.
// The value passed to the Await method must be a pointer or can be nil to ignore the returned value.
// Alternatively, tasks can be awaited using the task.WhenAll or task.WhenAny methods, allowing the workflow
// to block and wait for multiple tasks at the same time.
```

Reply:
Make no apologies for verbosity, thank you for clarifying!